src/cmakeProject.ts

```diff
@@ -123,6 +124,15 @@ export interface ConfigureCancelInformation {
     canceled: boolean;
 }
 
+/**
+ * A target with a name, but no output. Only used for targets that have a
+ * folder property.
+ */
+interface FolderTarget {
```

Comment:
Should this be defined in the same location as the RichTarget or NamedTarget? To keep things consistent?